jenkins-pipelines/manager-backup.jenkinsfile

```diff
@@ -12,6 +12,10 @@ longevityPipeline(
     test_name: 'mgmt_cli_test.MgmtCliTest.test_backup_feature',
     test_config: 'test-cases/manager/manager-regression-multiDC-set-distro.yaml',
 
+    scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo',
+    scylla_mgmt_agent_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo',
+    scylla_version: 'master:latest',
```

Comment:
shouldn't we use latest stable Scylla version instead of latest master?

Reply:
Well it's up to you guys really. for master it was like that

Reply:
I think that o.k. for those jobs, that what decided
On beach those can be overwritten

Reply:
the problem here will be that when running it daily, we might fail for Scylla issues, instead of focus on the Manager tests (as those are meant to test Manager).
@ShlomiBalalis , maybe you should create a PR to change these defaults

Reply:
that was @roydahan call, to run manager master on scylla master (on SCT master), master all the way down